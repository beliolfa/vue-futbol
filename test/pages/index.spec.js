import { createLocalVue, shallowMount, RouterLinkStub } from '@vue/test-utils'
import Vuex from 'vuex'
import IndexPage from '@/pages/index.vue'

const localVue = createLocalVue()

localVue.use(Vuex)

describe('IndexPage', () => {
  let store

  beforeEach(() => {
    store = new Vuex.Store({
      modules: {
        episodes: {
          state: {
            currentEpisode: null,
          },
        },
      },
    })
  })

  test('is a Vue instance', () => {
    const wrapper = shallowMount(IndexPage, {
      store,
      localVue,
      stubs: { NuxtLink: RouterLinkStub },
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
