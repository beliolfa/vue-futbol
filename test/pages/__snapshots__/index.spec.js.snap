// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage is a Vue instance 1`] = `
"<div class=\\"mx-auto flex-1 flex flex-col items-center justify-center\\">
  Index Page
</div>"
`;
